### PR TITLE
Preprod: movie: Terror from the Year 5000 (1958)

### PR DESCRIPTION
**Pull request by paul-sheridan**
This is from an era when one star, train wreck sci-fi films are still fun to watch. Rating 1/5.

### CODE REVIEW

**Comment by paul-sheridan** [comment]
Note I used *the dangers of unfettered scientific advancement* here with some reservation. My thinking here is to just toss in this movie for now with the proviso that we review *the dangers of unfettered scientific advancement* usages and make the them more objective later. The revised theme might capture the idea of well intentioned scientists getting in over their heads when their experiments lead to something bad that they didn't anticipate.

File: notes/stories/film/film-scifi-1950s.st.txt [hunk 1]
```diff
@@ -4074,6 +4074,49 @@ flying saucer [The Mysterians flew in a flying saucer.]
...
+time machine [A main novelty of the film is a large, immobile time machine that can transport people and objects from to and from the future.]
+fiancée and fiancée [Victor and Claire were engaged to be married.]
+father and daughter [Professor Erling and his adult daughter Claire were there throughout the story.]
+risk taking vs. playing it safe [There was the following ongoing conflict between Professor Erling and Victor: Victor was determined to plow ahead with testing out the time machine in spite of the risks while Erling urged they take a more cautious path involving additional testing and consulting with outside experts.]
+humanoid mutant [Victor inadvertently transported a murderous mutant from the year 5000 AD to the present.]
+existential risk to civilization [The people of the year 5000 AD were at risk of going extinct owing to the atmosphere being poisoned by radiation. The reason behind how the atmosphere had become irradiated was never spelled out.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-1950s.st.txt:4109]: Given the various hints and the cultural context of the 50s' America, I'd say we can read between the lines: 

https://www.themeontology.org/theme.php?name=nuclear%20holocaust

**Review by odinlake — COMMENTED** [review]
Comments inline...

File: notes/stories/film/film-scifi-1950s.st.txt [hunk 2]
```diff
@@ -4074,6 +4074,49 @@ flying saucer [The Mysterians flew in a flying saucer.]
...
+1958
+
+:: References
+https://en.wikipedia.org/wiki/Terror_from_the_Year_5000
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+the dangers of unfettered scientific advancement [A message of the story was that Victor's reckless fast tracking of the use of a prototype time machine directly resulted in a murderous mutant monster being transported from the year 5000 AD to the present day.]
```
[the remark this answers is not in the document]
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-1950s.st.txt:4103]: There was such talk, indeed. I'll leave as is.

File: notes/stories/film/film-scifi-1950s.st.txt [hunk 3]
```diff
@@ -4074,6 +4074,49 @@ flying saucer [The Mysterians flew in a flying saucer.]
...
+time machine [A main novelty of the film is a large, immobile time machine that can transport people and objects from to and from the future.]
```
[the remark this answers is not in the document]
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-1950s.st.txt]: Whoops, typo fixed.

File: notes/stories/film/film-scifi-1950s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-1950s.st.txt]: The leap is more-or-less made in the plot summary of film's Wikipedia article. So sure, a **nuclear holocaust**, why not?

**Pull request merged**